examples/compose/vtcompose/docker-compose.test.yml

```diff
@@ -29,6 +29,33 @@ services:
       - "8600"
     hostname: consul3
     image: consul:latest
+  external_db_host:
+    build:
+      context: ./external_db/mysql
```

Comment:
All the changes in this file are because it is being used in a test and the test was failing before. The file has been regenerated. Previous changes to vtcompose are also reflected here, because it was generated when those changes were made.